frontend/src/common/updatecheck.ts

```diff
@@ -0,0 +1,33 @@
+import AxiosStatic from 'axios';
+
+const axios = AxiosStatic.create();
+
+export class LatestVersionDto {
+  constructor(public stable: string, public beta: string) { }
+}
+
+class UpdatesService {
+  public async get(localVersion: string): Promise<LatestVersionDto> {
+    let config = {
+      headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': 'Hub/' + localVersion,
```

Comment:
Seems like a very old bug in Chrome: https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name

> **Note**
> The [User-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) header is no longer forbidden, [as per spec](https://fetch.spec.whatwg.org/#terminology-headers) — see forbidden header name list (this was implemented in Firefox 43) — it can now be set in a Fetch [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers) object, or via XHR [setRequestHeader()](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest#setrequestheader%28%29). However, Chrome will silently drop the header from Fetch requests (see [Chromium bug 571722](https://bugs.chromium.org/p/chromium/issues/detail?id=571722)).

I'm not sure if we should already do this in this PR but I think it's best to move the update checker into the backend.

Reply:
Ok, let's use `X-Hub-Version`. Strange behaviour, though...

Reply:
(This will also give us some insights into the actual user agents used)

Reply:
Changed in https://github.com/cryptomator/hub/pull/121/commits/9418fb52cde7872a7687ef20251e5633912cbd9c